src/components/form-fields/Error.component.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components/native';
+
+const ErrorText = styled.Text`
```

Comment:
better to extract intro error.styles.js

Reply:
Then if i want to customize for different screen i cant use inline styles